test/MasterViewController.m

```diff
@@ -40,21 +50,40 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
 {
-    return self.org.employees.count;
+    return self.org.employee.count;
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"myCell" forIndexPath:indexPath];
-    Employee *employee = self.org.employees[indexPath.row];
-    cell.textLabel.text = employee.fullName;
+    
+    NSSortDescriptor *firstNameDescriptor = [[NSSortDescriptor alloc] initWithKey:@"firstName" ascending:true];
```

Comment:
Add @property (readonly, nonatomic) NSArray<Employee *> sortedEmployees; Implement its logic and use it.